Adding constants for Kibibyte, Mebibyte, Gibibyte, etc.

barpavel opened the pull request:
Adding constants for Kibibyte, Mebibyte, Gibibyte, Tebibyte & Pebibyte and using them everywhere in tests.

Signed-off-by: Pavel Bar <pbar@redhat.com>

barpavel committed:
Adding constants for Kibibyte, Mebibyte, Gibibyte, Tebibyte & Pebibyte and using them everywhere in tests.

Signed-off-by: Pavel Bar <pbar@redhat.com>

nirs commented on tests/python_test.py:
```diff
@@ -256,9 +258,9 @@ def test_acquire_release_resource(tmpdir, sanlock_daemon, size, offset):
+    (KiB, sanlock.SECTOR_SIZE[0]),
     # Invalid sector size
-    (sanlock.ALIGN_SIZE[0], 8192),
+    (sanlock.ALIGN_SIZE[0], 8*KiB),
```
Missing space:

    8 * KiB

nirs commented on tests/units.py:
```diff
@@ -0,0 +1,10 @@
+GiB = 1024**3
+TiB = 1024**4
+PiB = 1024**5
+EiB = 1024**6
```
I don't think we use PiB or EiB, and we will probably never use them.

nirs commented on tests/daemon_test.py:
```diff
@@ -78,26 +79,26 @@ def test_init_resource(tmpdir, sanlock_daemon):
 def test_format(tmpdir, sanlock_daemon):
     path = tmpdir.join("rindex")
-    size = 1024**2 * 3
+    size = MiB * 3
```
While we fix this we should also change the order to:

    3 * MiB

Which is more natural.

nirs reviewed (changes requested):
Thanks, looks much nicer now!

2 issues:
- always put the count before the size (e.g. 3 * MiB instead of MiB * 3)
- keep space around *

barpavel commented:
1) Removed "EiB" constant, decided to leave "PiB" constant just in case we'll need it in some foreseeable future.
2) Changed the order when multiplying constant:
`MiB * 3 `--> `3 * MiB`
3) Added space around multiplication sign constant when multiplying constant:
`8*KiB` --> `8 * KiB`
Note: I decided to NOT add spaces in 2 places where this would not feel natural.
`def check_guard(path, size, guard=b"X", guard_size=4*KiB)`

nirs reviewed (commented):
Looks like duplicate of #11.

Please take the typos commit out of this PR.

barpavel commented:
Removed the typos commit.

nirs reviewed (approved):
Awesome, thanks!
